packages/calcite-components/src/assets/styles/includes.scss

```diff
@@ -229,3 +229,25 @@ $common-animatable-props: "background-color, block-size, border-color, box-shado
     font-weight: var(--calcite-font-weight-bold);
   }
 }
+
+@mixin input-placeholder-styles {
```

Comment:
To avoid unintentional usage of this, can you convert it to a [placeholder](https://sass-lang.com/documentation/style-rules/placeholder-selectors/) instead?